website/docusaurus.config.ts

```diff
@@ -40,7 +40,7 @@ const config: Config = {
 					lastVersion: 'current',
 					versions: {
 						current: {
-							label: '3.0.0-rc.4',
+							label: '3.0.0-rc.5',
```

Comment:
It doesn't need to precede this PR merging since its a non-breaking change, but the documentation will need to be updated to reflect increment.